mutiny-core/src/lib.rs

```diff
@@ -1308,6 +1310,22 @@ impl<S: MutinyStorage> MutinyWallet<S> {
         })
     }
 
+    pub async fn reissue_oob_notes(&self, oob_notes: OOBNotes) -> Result<bool, MutinyError> {
+        let federation_lock = self.federations.read().await;
+        let federation_ids = self.list_federation_ids().await?;
+
+        let maybe_federation_id = federation_ids
+            .iter()
+            .find(|id| id.to_prefix() == oob_notes.federation_id_prefix());
+
+        if let Some(fed_id) = maybe_federation_id {
+            let fedimint_client = federation_lock.get(&fed_id).ok_or(MutinyError::NotFound)?;
+            Ok(fedimint_client.reissue(oob_notes).await?)
+        } else {
+            return Err(MutinyError::NotFound);
+        }
```

Comment:
With the new version of OOB notes, they support having the federation code inside of them, right? I think it would be cool to do it like cashu where it joins the federation just to redeem it on lightning. 

https://github.com/fedimint/fedimint/pull/4231

Reply:
yep, we can have that when we bump to v0.3 😁

Reply:
Ah, didn't realized it wasn't backported.